### DISCUSSION TITLE
Complete Setup with Xiao Seeed Studio Esp32_c3 and GC9A01 Aliexpress version??

### DISCUSSION THREAD

**Discussion by EmrysCES** [Bodmer/TFT_eSPI#3132] [Compatible displays and setup files]
Dear Bodmer,

Thanks for the previous help! I recently watched this video:
https://www.youtube.com/watch?v=mqSe_uMpxIs&pp=ygUVdGhlbGFzdG91dHBvc3R3b2tzaG9w

The screen worked perfectly but I am now looking for a smaller chip. I tought the Xiao Seed studio ESP32_C3 chip would work well, but I am new to all of this. The things i want are pretty difficult and i think it would take a long time for you to figure out, so please think twice before helping me beacause they may be impossible. 

This is what i want:

1. The complete wireing from the Aliexpress GC9A01 screen with the Xiao Seed studio ESP32_C3 chip.
2.  The code in Arduino IDE for loading GIF files in the form of C header files onto the chip, similar to the video

Again, i know this is a lot of work, so feel free to just ignore my request if you feel like, but please tell me if you decide to not take on the task. It is pretty difficult and if you dont know how, would you please help me finding someone who has the skills neccesary?

Thank you for considering to help me and I would be very happy If you decide to help me.
Greetings,

EmrysCES

**Comment by jessicaengel451** [comment]
This is a good place to start. It does require quite a bit of technical knowledge. I am in the process of doing this with a variant of the screen that you have. https://github.com/thelastoutpostworkshop/ESP32ManyRoundScreenTest
